v8js_object_export.cc

```diff
@@ -1007,6 +1007,34 @@ static v8::MaybeLocal<v8::Object> v8js_wrap_object(v8::Isolate *isolate, zend_cl
 
 	v8::MaybeLocal<v8::Object> newobj = constr->NewInstance(v8_context, 1, &external);
 
+	bool has_json_serializable = false;
+    for (unsigned int i = 0; i < ce->num_interfaces; i ++) {
+        if (strcmp (ZSTR_VAL(ce->interfaces[i]->name), "JsonSerializable") == 0) {
+            has_json_serializable = true;
+            break;
+        }
+    }
+
+    if (has_json_serializable) {
+        void *ptr;
+        zend_string *key;
+        ZEND_HASH_FOREACH_STR_KEY_PTR(&ce->function_table, key, ptr) {
+            zend_function *method_ptr = reinterpret_cast<zend_function *>(ptr);
+            if (strcmp(ZSTR_VAL(method_ptr->common.function_name), "jsonSerialize") == 0) {
```

Comment:
The foreach loop combined with `strcmp` effectively is just a hash table lookup, isn't it?
`zend_hash_find_ptr` should do just that more efficiently.

Reply:
Yes! This foreach is the same thing.
But I don't know why `zend_has_find_ptr` didn't worked. I've tried a lot with no luck.
I will try again with "cold head" 🤣